csi/util.go

```diff
@@ -239,6 +242,33 @@ func parseJSONRecurringJobs(jsonRecurringJobs string) ([]longhornclient.Recurrin
 	return recurringJobs, nil
 }
 
+func syncMountPointDirectory(targetPath string) error {
+	d, err := os.OpenFile(targetPath, os.O_SYNC, 0750)
+	if err != nil {
+		return err
+	}
+	if _, err := d.Readdirnames(1); err != nil {
+		if !errors.Is(err, io.EOF) {
+			d.Close()
+			return err
+		}
+	}
+	if err := d.Close(); err != nil {
```

Comment:
nit: you can actually do the defer d.close after it has been successfully opened, but return error in the defer function if you want.

Reply:
Modified. Thanks